examples/input-artifact-rawcontent.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: input-artifact-raw-
+spec:
+  entrypoint: raw-contents
+  templates:
+  - name: raw-contents
+    inputs:
+      artifacts:
+      - name: myfile
+        path: /tmp/file
+        raw:
+          contents: |
```

Comment:
Lets check with @edlee2121 and folks on syntax. I'm now thinking we should use the term "data" to keep in line with ConfigMaps and Secrets.

Reply:
Let's use the terminology most consistent with k8s.

Reply:
ok, will use "data"